frontend/src/app/components/crm/leads/leads.component.ts

```diff
@@ -24,8 +26,6 @@ import { first } from 'rxjs/operators';
 
 import { Lead } from '../../../models/server_responses/lead.model';
 import { LeadService } from '../../../services/lead.service';
-import { Title } from '@angular/platform-browser';
-import * as _ from 'lodash';
```

Comment:
Don't delete these. These are used for displaying title at the top of the page.

Reply:
umm my bad